modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc

```diff
@@ -40,7 +40,7 @@ spec:
 # ...
 ----
 
-. Optional: Enter the following command to confirm that the Operator has completed provisioning. This might take a few minutes. 
+. Optional: Enter the following command to confirm that the Operator has completed provisioning. This might take a few minutes.
```

Comment:
Might sound silly but "This might take a few minutes" makes it read like entering the command takes a few minutes. How about:

> You might need to wait a few minutes for the command to complete.

Reply:
@dfitzmau This PR didn't modify this lines - this is just whitespace trimming. I'm just updating the commands